src/Apps/Artist/ArtistApp.tsx

```diff
@@ -65,14 +66,15 @@ export class ArtistApp extends React.Component<ArtistAppProps> {
           </Col>
         </Row>
 
-        {me && (
-          <>
-            <Separator my={6} />
-            <LazyLoadComponent threshold={300}>
-              <RecentlyViewed me={me as any} />
-            </LazyLoadComponent>
-          </>
-        )}
+        {me &&
+          typeof window !== "undefined" && (
```

Comment:
@zephraph - Note that as of right now `<LazyLoadComponent>` requires a `window`; I'm gonna open a PR over there to make it safer.